.travis.yml

```diff
@@ -153,7 +153,7 @@ before_script:
       javac -classpath jcardsim/target/jcardsim-3.0.5-SNAPSHOT.jar IsoApplet/src/net/pwendland/javacard/pki/isoapplet/*.java;
       echo "com.licel.jcardsim.card.applet.0.AID=F276A288BCFBA69D34F31001" > isoapplet_jcardsim.cfg;
       echo "com.licel.jcardsim.card.applet.0.Class=net.pwendland.javacard.pki.isoapplet.IsoApplet" >> isoapplet_jcardsim.cfg;
-      echo "com.licel.jcardsim.card.ATR=3B80800101" >> isoapplet_jcardsim.cfg;
+      echo "com.licel.jcardsim.card.ATR=3B8B8001805949736F4170706C6574BA" >> isoapplet_jcardsim.cfg;
```

Comment:
Why do you choose this ATR? I think, IsoApplet doesn't require any specific one...

Reply:
I chose this specific ATR for no reason, any unique one would do. 
As I explained in my answer to @Jakuje, If you leave the stock ATR, then you can not initialize the token using _PKCS#11_ functions (C_InitToken & C_InitPin) as `pkcs11_enable_InitToken` is taken from the `card_atr 3B:80:80:01:01` section which "belongs" to one of the SmartCard-HSMs, unless of course you add  `pkcs11_enable_InitToken=yes` to that section...